pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -59,7 +59,7 @@ in stdenv.mkDerivation ({
 
   debugInfo = enableDebugInfo;
 
-  enableParallelBuilding = true;
+  enableParallelBuilding = false;
```

Comment:
Please add a brief comment here, why it was disabled. Otherwise someone might enable it again.

Reply:
Thanks, added.